sys/netbsd/sys.txt

```diff
@@ -190,6 +194,46 @@ ucred {
 	gid	gid
 }
 
+posix_spawn_file_actions {
+	size	int32
+	len	int32
+	fae	ptr[in, posix_spawn_file_actions_entry]
+}
+
+posix_spawn_file_actions_entry {
+	fae_action	int32
+	fae_filedes	int32
+	fae_data	fae_data_t
+}
+
+fae_data_t [
+	open	open_t
+	dup2	dup2_t
+]
+
+open_t {
+	path	ptr[in, string]
+	oflag	int32
+	mode	int32
```

Comment:
I guess this is some flags, and we already should have them described for open syscall.